spotifyMini/ViewControllers/Analysis/GenreSelectTableViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  GenreSelectTableViewController.swift
+//  spotifyMini
+//
+//  Created by Tom OMalley on 5/17/16.
+//  Copyright © 2016 intrepid. All rights reserved.
+//
+
+import Foundation
+
+class GenreSelectTableViewController : UITableViewController {
+
+    let cellIdentifier = "genreCell"
```

Comment:
So I would need to, in storyboard, set the cell identifier to be "UITableViewCell" ? or  "GenreSelectTableViewController" ?